### PR TITLE
Add storage permissions to options page example

### PR DESCRIPTION
**Pull request by Florian-COLLIN**
Changes proposed in this pull request:

- Added storage permission for the Google Chrome extension options section.

### CODE REVIEW

**Comment by dotproto** [comment]
@Florian-COLLIN, in order for us to accept this change you will need to sign the Contributor License Agreement (CLA). See [this build error](https://github.com/GoogleChrome/developer.chrome.com/pull/1957/checks?check_run_id=4817656009) for details.

**Comment by Florian-COLLIN** [comment]
Normally it's good. 

Regards, 

Florian COLLIN

File: site/en/docs/extensions/mv3/options/index.md [hunk 1]
```diff
@@ -79,6 +79,19 @@ document.getElementById('save').addEventListener('click',
     save_options);
 ```
 
+For this to work, you must add in the `manifest.json`:
+
+```json/3
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/options/index.md:84]:
```suggestion
```json/4
```

Highlight the "storage" permission in the code block.

File: site/en/docs/extensions/mv3/options/index.md [hunk 2]
```diff
@@ -79,6 +79,19 @@ document.getElementById('save').addEventListener('click',
     save_options);
 ```
 
+For this to work, you must add in the `manifest.json`:
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/options/index.md:82]:
```suggestion
Finally, add the `"storage"` permission to the extension's `manifest.json`:
```

**Review by dotproto — CHANGES_REQUESTED** [review]
Tweak framing and code sample.

**Review by dotproto — APPROVED** [review]

**Pull request merged**